### PR TITLE
Option to skip extract step

### PR DESCRIPTION
**Pull request by henryjw**
Requested by #11

## Example screenshot
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +67,7 @@ jobs:
         with:
           cache-source: var-lib-apt
           cache-target: /var/lib/apt
+          skip-extraction: ${{ steps.cache-var-lib-apt.outputs.cache-hit }}
```
Comment (AkihiroSuda) [inline README.md:70]: This has to be a separate PR, as `skip-extraction` is missing in `buildkit-cache-dance@v2.1.3`

**Comment by AkihiroSuda** [comment]
Thanks, but please sign the commit for DCO
https://github.com/apps/dco

(run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

**Comment by AkihiroSuda** [comment]
Also please squash commits

File: README.md (hunk 1 above)
Comment (henryjw) [reply README.md]: Done ✅

**Comment by henryjw** [comment]
> Thanks, but please sign the commit for DCO [github.com/apps/dco](https://github.com/apps/dco)
> 
> (run `git commit -a -s --amend`, and make sure that the `Signed-off-by: NAME <EMAIL>` line with your real name is included in the commit message)

Done ✅

**Review by AkihiroSuda — APPROVED** [review]
Thanks
